conda_forge_tick/migrators/arch.py

```diff
@@ -413,3 +413,150 @@ def pr_body(self, feedstock_ctx: ClonedFeedstockContext) -> str:
+                _cut_to_target_packages(graph, target_packages)
+
+            # filter out stub packages and ignored packages
+            _filter_stubby_and_ignored_nodes(graph, self.ignored_packages)
+
+        if not hasattr(self, "_init_args"):
+            self._init_args = []
+
+        if not hasattr(self, "_init_kwargs"):
+            self._init_kwargs = {
+                "graph": graph,
+                "name": name,
+                "pr_limit": pr_limit,
+                "piggy_back_migrations": piggy_back_migrations,
+                "target_packages": target_packages,
+                "effective_graph": effective_graph,
+                "_do_init": False,
+            }
+
+        super().__init__(
+            graph=graph,
+            pr_limit=pr_limit,
+            check_solvable=False,
+            piggy_back_migrations=piggy_back_migrations,
+            effective_graph=effective_graph,
+        )
+
+        assert not self.check_solvable, "We don't want to check solvability for aarch!"
+        self.target_packages = target_packages
+        self.name = name
+
+        if _do_init:
+            self._reset_effective_graph()
+
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        if super().filter(attrs):
+            return True
+        muid = frozen_to_json_friendly(self.migrator_uid(attrs))
+        for arch in self.arches:
+            configured_arch = (
+                attrs.get("conda-forge.yml", {}).get("provider", {}).get(arch)
+            )
+            if configured_arch:
+                return muid in _sanitized_muids(
+                    attrs.get("pr_info", {}).get("PRed", []),
+                )
+        else:
+            return False
+
+    def migrate(
+        self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any
+    ) -> "MigrationUidTypedDict":
+        with pushd(recipe_dir + "/.."):
+            self.set_build_number("recipe/meta.yaml")
+            with open("conda-forge.yml") as f:
+                y = yaml_safe_load(f)
+            if "provider" not in y:
+                y["provider"] = {}
+            for k, v in self.arches.items():
+                if k not in y["provider"]:
+                    y["provider"][k] = v
+
+            with open("conda-forge.yml", "w") as f:
+                yaml_safe_dump(y, f)
+
+        return super().migrate(recipe_dir, attrs, **kwargs)
+
+    def pr_title(self, feedstock_ctx: FeedstockContext) -> str:
+        return "Windows ARM Migrator"
+
+    def pr_body(self, feedstock_ctx: ClonedFeedstockContext) -> str:
+        body = super().pr_body(feedstock_ctx)
+        body = body.format(
+            dedent(
+                """\
+        This feedstock is being rebuilt as part of the windows arm migration.
+
+        **Feel free to merge the PR if CI is all green, but please don't close it
+        without reaching out the the ARM Windows team first at <code>@</code>conda-forge/help-win-arm64.**
```

Comment:
Does this team exist? 🤔

Reply:
Yes we should make this team too! We also need people to be on the team. 🤔

Reply:
I and @baszalmstra are surely happy to be on the team :)

Reply:
For sure!

Reply:
Teams are only for project members, right? If not, feel free to add @finnagin as well (or else just ping him on anything that comes up)

Reply:
We'll need to make this team outside of the normal staged-recipes subteams unless we vote all of the members onto staged-recipes.